tests/topotests/lib/topotest.py

```diff
@@ -1318,8 +1318,8 @@ def fix_netns_limits(ns):
     if version_cmp(platform.release(), "4.20") >= 0:
         sysctl_assure(ns, "net.ipv6.route.skip_notify_on_dev_down", 1)
 
-    sysctl_assure(ns, "net.ipv4.conf.all.ignore_routes_with_linkdown", 1)
-    sysctl_assure(ns, "net.ipv6.conf.all.ignore_routes_with_linkdown", 1)
+    sysctl_assure(ns, "net.ipv4.conf.default.ignore_routes_with_linkdown", 1)
```

Comment:
We have this commit here https://github.com/FRRouting/frr/pull/15633/commits/cefebd87c1a9fc672cb26653cbb7806bc4e655af...

Reply:
removed.